template.tpl

```diff
@@ -0,0 +1,277 @@
+    "help": "Your Chartbeat UID (Account ID). See \u003ca href\u003d\"http://support.chartbeat.com/docs/#accounts\"\u003eChartbeat standard configuration parameters\u003c/a\u003e.",
+    "alwaysInSummary": true,
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ]
+  },
+  {
+    "type": "TEXT",
+    "name": "domain",
+    "displayName": "domain",
+    "simpleValueType": true,
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
+    "help": "Domain parameter. See \u003ca href\u003d\"http://support.chartbeat.com/docs/#code\"\u003eChartbeat standard configuration parameters\u003c/a\u003e.",
+    "alwaysInSummary": true
+  },
+  {
+    "type": "TEXT",
+    "name": "sections",
+    "displayName": "sections",
+    "simpleValueType": true,
+    "help": "Comma-separated list of sections. See \u003ca href\u003d\"http://support.chartbeat.com/docs/#sectionsauthors\"\u003eChartbeat standard configuration parameters\u003c/a\u003e and \u003ca href\u003d\"https://www.google.com/url?q\u003dhttps://support.google.com/tagmanager/answer/6164391?hl%3Den\u0026sa\u003dD\u0026ust\u003d1593065281792000\u0026usg\u003dAFQjCNGIkL6gneEEH0OkcfEw18Nx1z3D7A\"\u003eData Layers\u003c/a\u003e."
+  },
+  {
+    "type": "TEXT",
+    "name": "authors",
+    "displayName": "authors",
+    "simpleValueType": true,
+    "help": "Comma-separated list of authors. See \u003ca href\u003d\"http://support.chartbeat.com/docs/#sectionsauthors\"\u003eChartbeat standard configuration parameters\u003c/a\u003e and \u003ca href\u003d\"https://www.google.com/url?q\u003dhttps://support.google.com/tagmanager/answer/6164391?hl%3Den\u0026sa\u003dD\u0026ust\u003d1593065281792000\u0026usg\u003dAFQjCNGIkL6gneEEH0OkcfEw18Nx1z3D7A\"\u003eData Layers\u003c/a\u003e."
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "flickerControl",
+    "checkboxText": "Enable Headline Testing Flicker Control?",
+    "simpleValueType": true,
+    "help": "By default this is set to false. If you are using our Headline Testing tool and need to implement Flicker Control, please review our \u003ca href\u003d\"http://support.chartbeat.com/docs/headlinetesting.html\"\u003edocumentation\u003c/a\u003e before enabling this feature."
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+// Enter your template code here.
+const log = require('logToConsole');
+const query = require('queryPermission');
+const makeString = require('makeString');
+const injectScript = require('injectScript');
+const setInWindow = require('setInWindow');
+const copyFromWindow = require('copyFromWindow');
+const queryPermission = require('queryPermission');
+const createQueue = require('createQueue');
+if (queryPermission('access_globals', 'readwrite', 'dataLayer')) {
+  const dataLayerPush = createQueue('dataLayer');
+}
+
+log('data =', data);
+
+const uid = data.uid;
+const domain = data.domain;
+const sections = data.sections;
+const authors = data.authors;
+const flickerControl = data.flickerControl;
```

Comment:
If you are not using these const other than to set the `_sf_async_config` later... I would say just use 
```
_sf_async_config.uid = data.uid;
_sf_async_config.domain = data.domain; 
_sf_async_config.flickerControl = data.flickerControl;
_sf_async_config.sections = data.sections; 
_sf_async_config.authors = data.authors;
```